wwwroot/index.html

```diff
@@ -507,10 +765,10 @@ <h4 class="card-title">Max Hillebrand</h4>
 		<footer>
 			Legal notice:
 			<a class="go-to ml-1 mr-1"
-				href="https://zksnacks.com/wp-content/uploads/2018/05/zkSNACKs_Privacy-Policy_vfinal.pdf"
+				href="https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi/Legal/Assets/LegalDocuments.txt"
 				target="_blank"><span style="font-weight: 400;">Privacy policy</span></a>
 			<span class="go-to"> | </span>
-			<a class="go-to ml-1" href="https://zksnacks.com/wp-content/uploads/2018/05/zkSNACKs_Legal.pdf"
+			<a class="go-to ml-1" href="mailto:legal@zksnacks.com"
```

Comment:
@bharmat @hgergoil old file was missing. What if we add `mailto:legal@zksnacks.com`?